### PR TITLE
Sonar scanner dotnet

### PR DESCRIPTION
**Pull request by dustinvanbuskirk**
@brandonphillips Can I get a review please?

### CODE REVIEW

**Comment by dustinvanbuskirk** [comment]
[Step for DOTNET C# C++ Sonarqube](https://trello.com/c/YC0wGQLM/32-step-for-dotnet-c-c-sonarqube)

File: incubating/sonar-scanner-dotnet-linux/step.yaml [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+        },
+        "PRE_COMMANDS": {
+          "type": "array",
+          "description": "Commands to execute before scan command."
+        },
+        "POST_COMMANDS": {
+          "type": "array",
+          "description": "Commands to execute after scan command."
+        },
+        "WORKING_DIRECTORY": {
```
Comment (brandonphillips) [inline incubating/sonar-scanner-dotnet-linux/step.yaml:123]: Should this be a required field?

Comment (dustinvanbuskirk) [reply incubating/sonar-scanner-dotnet-linux/step.yaml:123]: Nope, if you look below this is just a way to specify a path deeper than the root of the checkout.  The checkout is default.

File: incubating/sonar-scanner-dotnet-linux/step.yaml [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+        "SONAR_HOST_URL",
+        "SONAR_LOGIN",
+        "SONAR_PROJECT_KEY"
+      ],
+      "properties": {
+        "SONAR_HOST_URL": {
+          "type": "string",
+          "description": "Sonarqube Server Host URL."
+        },
+        "SONAR_HOST_URL": {
```
[the remark this answers is not in the document]
Comment (dustinvanbuskirk) [reply incubating/sonar-scanner-dotnet-linux/step.yaml]: Fixed.

**Review by brandonphillips — APPROVED** [review]

**Pull request merged**